CNAME

```diff
@@ -1 +0,0 @@
-meanjs.org
```

Comment:
I made that change after I submitted the pull request to make it render on my GitHub Pages for demonstration. I forgot that it would have added it too the pull request. My apologies. Once it's approved too be merged let me know and I'll change it back.